app/src/main/java/com/example/areumdap/UI/Task/TaskViewModel.kt

```
@@ -131,35 +129,25 @@ class TaskViewModel(private val apiService: TaskApiService) : ViewModel() {
-
                         questionsNextCursorTime = data.nextCursorTime
                         questionsNextCursorId = data.nextCursorId
                         _questionsHasNext.value = data.hasNext
                     }
                 } else {
                     val errorBody = response.errorBody()?.string() ?: ""
-                    Log.e("TaskViewModel", "Question Server Error: ${response.code()}, Body: $errorBody")
+
                     _errorMessage.value = "서버 에러(${response.code()}): $errorBody"
                 }
             } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)